src/NHSD.BuyingCatalogue.Identity.Api/Infrastructure/TagHelperConstants.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace NHSD.BuyingCatalogue.Identity.Api.Infrastructure
+{
+    public static class TagHelperConstants
+    {
+        // tags
+        public const string Div = "div";
+        public const string Anchor = "a";
+        public const string Link = "href";
+        public const string ListItem = "li";
+        public const string SubHeader = "h2";
+        public const string Span = "span";
+        public const string UnorderedList = "ul";
+
+        // attributes
+        public const string DataTestId = "data-test-id";
+        public const string For = "asp-for";
+        public const string Id = "id";
+        public const string LabelledBy = "aria-labelledy";
+        public const string Role = "role";
+        public const string RoleAlert = "alert";
+        public const string Type = "type";
+
+        // classes
+        public const string ErrorSummaryTitle = "error-summary-title";
+        public const string NhsErrorMessage = "nhsuk-error-message";
+        public const string NhsFormGroup = "nhsuk-form-group";
+        public const string NhsFormGroupError = "nhsuk-form-group--error";
+        public const string NhsInput = "nhsuk-input";
+        public const string NhsLabel = "nhsuk-label";
+        public const string NhsList = "nhsuk-list";
+        public const string NhsValidationSummary = "nhsuk-error-summary";
+        public const string NhsValidationSummaryTitle = "nhsuk-error-summary__title";
+        public const string NhsValidationSummaryList = "nhsuk-error-summary__list";
+    }
+}
```

Comment:
`internal` accessibility is probably enough for the class and all constants here but public constants obviously aren't going to do any harm.